src/command/supergraph/compose/do_compose.rs

```diff
@@ -186,131 +215,11 @@ impl Compose {
 
 #[cfg(test)]
 mod tests {
-    use std::convert::TryFrom;
```

Comment:
These tests have now been moved, and centralised in `utils/supergraph_config.rs`